src/Symfony/Component/Lock/LockableResourceInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Lock;
+
+/**
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ */
+interface LockableResourceInterface
+{
+    public function getResourceIdentifier(): string;
```

Comment:
getLockIdentifier to make the domain clear?

Reply:
or getLockKey?

Reply:
`getLockResource` would actually be the correct one based on the Lock domain.

This method does not return a key.